hummingbot/client/hummingbot_application.py

```diff
@@ -1141,6 +1142,99 @@ def compare_balance_snapshots(self):
         lines = ["", "  Performance:"] + ["    " + line for line in str(df).split("\n")]
         self._notify("\n".join(lines))
 
+    def add_balances(self, market_symbol_pair, is_base, is_starting, base, quote):
+        """ Adds the balance of either the base or the quote in the given market symbol pair token to the corresponding
+        tuple pair.
+
+        :param market_symbol_pair: type market_symbol_pair
+        :param is_base: boolean indicating whether to add base amount of token or quote
+        :param is_starting: boolean indicating whether to calculating starting amount of token or current
+        :param base: tuple that is the base currency token pair
+        :param quote: tuple that is the quote currency token pair
+        """
+        market_name = market_symbol_pair.market.name
+
+        if is_base:
+            asset_tuple = base
+            asset = market_symbol_pair.base_asset
+        else:
+            asset_tuple = quote
+            asset = market_symbol_pair.quote_asset
+
+        if is_starting:
+            amount = self.starting_balances.get(asset).get(market_name)
```

Comment:
If you're sure the asset or market name exists in the dicts, you should just use `[]` instead of `get()`. `get()` should only be used when you're not sure and you want a sensible default value.